### ISSUE TITLE
Ensure user secrets are not logged.

### ISSUE THREAD

**Issue by adunkman** [ustaxcourt/ef-cms#283]
As the Court, so that we can maintain a secure system, we need to avoid logging information which may allow a log consumer from assuming the role of another user.

Acceptance criteria

- JWT tokens are not logged.
- When a user is logged, only [what specifically allowed attributes?] are logged. 
- Any data sent outside of AWS, including to services like Honeybadger, does not contain personally identifiable information. 

Also tracked at Flexion’s Production Prep items: 
- [Logging Security - High](https://trello.com/c/99udrlur/582-logging-security-high)
- [HoneyBadger Logging - Low](https://trello.com/c/fTcLbTHu/586-honeybadger-logging-low)

**Comment by adunkman** [comment]
Groom with ustaxcourt/ef-cms-flexion#234.

**Comment by adunkman** [comment]
We’ll need to clarify what is acceptable to write into logs — including PII, and what kinds. A useful artifact here will be what is acceptable to be written in logs — this should be documented in an accessible place to guide future development.

**Comment by adunkman** [comment]
This issue will focus on low-hanging fruit, since we’ve already decided to not ship these logs to third parties outside of AWS.

**Comment by vickimcfadden** [comment]
Blocked by [#6408](https://github.com/flexion/ef-cms/issues/6408) in Flexion backlog.

**Comment by vickimcfadden** [comment]
Pre-MVP, access tokens and header.authorization bearer tokens. For anything else, we'll open a new ticket for post-MVP.

**Comment by adunkman** [comment]
ustaxcourt/ef-cms-flexion#502 implemented filtering out access tokens and header.authorization bearer tokens, no data is sent to external services like Honeybadger from the Court’s environments. 

https://github.com/flexion/ef-cms/issues/6408 also includes some comments about AWS tokens, so that ticket likely includes additional work not captured here. 

> Pre-MVP, access tokens and header.authorization bearer tokens. For anything else, we'll open a new ticket for post-MVP.

This ticket is now complete.
